D2Go/ops/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.4.1)
```

Comment:
I believe this file is only necessary if we don't use torchvision pre-compiled artifacts directly from jcenter, correct? In this case, might be good to add a comment on the top of it indicating that this is only if one wants to build torchvision themselves.

Reply:
Yes, we do not need ops folder
the user just adds gradle dependency 

implementation 'org.pytorch:pytorch_android_torchvision:1.8.0-SNAPSHOT'

and 

NativeLoader.loadLibrary("torchvision_ops");

in java code to load that library

Reply:
Done.